test/codegen/fromcto/vocabulary/vocabularyvisitor.js

```diff
@@ -464,4 +464,29 @@ describe('VocabularyVisitor', function () {
             param.fileWriter.writeLine.withArgs(0,'      - VALUE: VALUE of the Dictionary').calledOnce.should.be.ok;
         });
     });
-});
+
+    describe('visitProperty for Field', () => {
+        let param;
+        beforeEach(() => {
+            param = {
+                fileWriter: mockFileWriter
+            };
+        });
+        
+        it('should skip vocabulary generation for system property', () => {
```

Comment:
Please move the test [this ](https://github.com/accordproject/concerto-codegen/blob/f0b85700fea0c1092f76cd2a816251b074060863/test/codegen/fromcto/vocabulary/vocabularyvisitor.js#L369) test suite for fields.
We already have a test suite for `visitProperty for Field`

Reply:
I modified both files accordingly. I believe it should be fixed now. Thanks for your patience.